testsuite-native-image/pom.xml

```diff
@@ -70,7 +70,7 @@
         </property>
       </activation>
       <properties>
-        <skipNativeImageTestsuite>true</skipNativeImageTestsuite>
+        <skipNativeImageTestsuite>false</skipNativeImageTestsuite>
```

Comment:
I don't believe this is correct. Skip tests should also skip the native test suite.

Reply:
+1 to @pmlopes comment